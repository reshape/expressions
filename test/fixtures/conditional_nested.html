<p>x</p>
<if condition='true'>
  <p>y</p>
  <if condition='true'>
    <p>z</p>
  </if>
  <p>y</p>
</if>
<p>x</p>
