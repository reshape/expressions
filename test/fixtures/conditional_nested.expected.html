<p>x</p>

  <p>y</p>
  
    <p>z</p>
  <p>y</p>
<p>x</p>
